l10n_es_aeat_verifactu/models/verifactu_mixin.py

```diff
@@ -100,6 +118,63 @@ def _compute_verifactu_macrodata(self):
                 >= 0
             )
 
+    def _compute_verifactu_qr_url(self):
+        """Returns the URL to be used in the QR code. A sample URL would be (urlencoded):
+        https://prewww2.aeat.es/wlpl/TIKECONT/ValidarQR?nif=89890001K&numserie=12345678%26G33&fecha=01-01-2024&importe=241.4
+        """
+        for record in self:
+            agency = self.env.ref("l10n_es_aeat.aeat_tax_agency_spain")
+            if record.company_id.verifactu_test:
+                qr_base_url = agency.verifactu_qr_base_url_test_address
+            else:
+                qr_base_url = agency.verifactu_qr_base_url
+
+            # Calc QR URL values
+            company_vat = self.company_id.vat
+            if company_vat and company_vat.startswith("ES"):
+                company_vat = company_vat[2:]
```

Comment:
en el módulo l10n_es_aeat disponemos de una función que ya devuelve el nif en el formato correcto
https://github.com/ozono/l10n-spain/blob/9794ab52202dcbab2fcdb117327a5b86361dd7ff/l10n_es_aeat/models/res_partner.py#L93
```suggestion
            company_vat = record.company_id.partner_id._parse_aeat_vat_info()[2]
```